networkx/readwrite/tests/test_gexf.py

```diff
@@ -498,3 +500,113 @@ def test_specials(self):
         assert H.nodes[1]['networkx_key'] == 'a'
         assert H.nodes[2]['networkx_key'] == 'b'
         assert H.nodes[3]['networkx_key'] == 'c'
+
+    def test_simple_list(self):
+        G = nx.Graph()
+        list_value = [(1, 2, 3), (9, 1, 2)]
+        G.add_node(1, key=list_value)
+        fh = io.BytesIO()
+        nx.write_gexf(G, fh)
+        fh.seek(0)
+        H = nx.read_gexf(fh, node_type=int)
+        assert H.nodes[1]['networkx_key'] == list_value
+
+    def test_dynamic_mode(self):
+        G = nx.Graph()
+        G.add_node(1, label='1', color='green')
+        G.graph['mode'] = 'dynamic'
+        fh = io.BytesIO()
+        nx.write_gexf(G, fh)
+        fh.seek(0)
+        H = nx.read_gexf(fh, node_type=int)
+        assert sorted(G.nodes()) == sorted(H.nodes())
+        assert (
+            sorted(sorted(e) for e in G.edges()) ==
+            sorted(sorted(e) for e in H.edges()))
+
+    def test_multigraph_with_missing_attributes(self):
+        G = nx.MultiGraph()
+        G.add_node(0, label='1', color='green')
+        G.add_node(1, label='2', color='green')
+        G.add_edge(0, 1, id='0', weight=3, type='undirected', start=0, end=1)
+        G.add_edge(0, 1)
+        fh = io.BytesIO()
+        nx.write_gexf(G, fh)
+        fh.seek(0)
+        H = nx.read_gexf(fh, node_type=int)
+        assert sorted(G.nodes()) == sorted(H.nodes())
+        assert (
```

Comment:
I would write this as follows:
```
        assert (
            sorted(sorted(e) for e in G.edges()) ==
            sorted(sorted(e) for e in H.edges())
        )
```

Reply:
If I understand correctly, you want me to change the indentation of the line to the style you mentioned, and I would be glad to do that; however, in the existing code, the style I followed is prevalent (eg: in `test_default_attribute`), so should I refractor them too or only make changes in my part?

Reply:
Note: Utilizing that style seems to be against **pep8** standards 😅

Reply:
You are correct.  Let's break before the binary operator:
```
assert (sorted(sorted(e) for e in G.edges())
        == sorted(sorted(e) for e in H.edges()))
```
and you show do the same thing in the other locations as well.